src/AiidaDFTK.jl

```diff
@@ -128,9 +129,13 @@ function run_json(filename::AbstractString; extra_output_files=String[])
     # Threading setup ... maybe later need to take parameters
     # from the JSON into account
     if mpi_nprocs() > 1
-        disable_threading()
-    else
-        setup_threading()
+        # Julia threads are controlled when launching Julia and should be disabled by the user.
+        if Threads.nthreads() > 1
+            @warn("Running through MPI, but threading was not disabled!"
+                + " Julia threads: $(Threads.nthreads()).")
+        end
+        # BLAS threads are controlled separately and can be disabled at runtime.
+        BLAS.set_num_threads(1)
```

Comment:
Interestingly, no, see https://carstenbauer.github.io/ThreadPinning.jl/v0.7/explanations/blas/

Reply:
What about FFTW threads ? Maybe just run `setup_threading(n_blas=1)` ?

Reply:
I changed it to set both through DFTK now.